### DISCUSSION TITLE
[A-L] (2023/24) Foglio 4 - Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#156] [Esercizi - Bertini]
![277413569-3386136f-30a9-4db3-b007-a913c05df7f9](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/a5ad439e-961b-4423-a5c8-4bc2d41b8219)

**Comment by Elia-Belli** [comment]
Per ipotesi: $\mathbb{P}(A \cap B)=\mathbb{P}(A)\mathbb{P}(B)$ ; $\mathbb{P}(A \cap C)= \mathbb{P}(A)\mathbb{P}(C)$ ; $\mathbb{P}(B\cap C)= \mathbb{P}(B)\mathbb{P}(C)$ ; $\mathbb{P}(A \cap B \cap C)=\mathbb{P}(A)\mathbb{P}(B)\mathbb{P}(C)$

- **Proposizione 1**: in generale se $A,B$ sono indipendenti, allora anche $A^c,B$ lo sono:
**Dim**: per ipotesi $\mathbb{P}(A|B)=\frac{\mathbb{P}(A \cap B)}{\mathbb{P}(B)}=\mathbb{P}(A)$
$A^c,B$ indipendenti $\iff \mathbb{P}(A^c|B)=\mathbb{P}(A^c)\ \land\ \mathbb{P}(B|A^c)=\mathbb{P}(B)$
$\frac{\mathbb{P}(A^c\cap B)}{\mathbb{P}(B)}=1-\mathbb{P}(A)\rightarrow \mathbb{P}(A^c\cap B)=\mathbb{P}(B)-\mathbb{P}(A)\mathbb{P}(B)= \mathbb{P}(B)(1-\mathbb{P}(A))=\mathbb{P}(B)\mathbb{P}(A^c)\ \checkmark$
$\frac{\mathbb{P}(A^c\cap B)}{\mathbb{P}(A^c)}=\mathbb{P}(B)\rightarrow \mathbb{P}(A^c\cap B)=\mathbb{P}(B)\mathbb{P}(A^c)\checkmark$

- **Proposizione 2**: in generale se $A,B$ sono indipendenti, allora anche $A^c,B^c$ lo sono:
**Dim**: $A^c,B^c$ indipendenti $\iff \mathbb{P}(A^c|B^c)=\mathbb{P}(A^c)\ \land\ \mathbb{P}(B^c|A^c)=\mathbb{P}(B^c)$
$\iff \frac{\mathbb{P}(A^c\cap B^c)}{\mathbb{P}(B^c)}=\mathbb{P}(A^c)\rightarrow\mathbb{P}(A^c\cap B^c)=\mathbb{P}(A^c)\mathbb{P}(B^c)\ \checkmark$
$\iff \frac{\mathbb{P}(A^c\cap B^c)}{\mathbb{P}(A^c)}=\mathbb{P}(B^c)\rightarrow\mathbb{P}(A^c\cap B^c)=\mathbb{P}(B^c)\mathbb{P}(A^c)\ \checkmark$

- **Proposizione 3**: in generale se $A,B,C$ sono indipendenti, allora anche $A^c,B,C$ lo sono:
**Dim**: $A^c,B,C$ indipendenti $\iff \mathbb{P}(A^c|(B\cap C))=\mathbb{P}(A^c),\mathbb{P}(B|(A^c\cap C))=\mathbb{P}(B),\mathbb{P}(C|(B\cap A^c))=\mathbb{P}(C)$
$\frac{\mathbb{P}(A^c\cap B \cap C)}{\mathbb{P}(B\cap C)}=\mathbb{P}(A^c)\rightarrow\mathbb{P}(A^c\cap B \cap C)=\mathbb{P}(A^c)\mathbb{P}(B \cap C)=\mathbb{P}(A^c)\mathbb{P}(B)\mathbb{P}(C)\ \checkmark$
$\frac{\mathbb{P}(A^c\cap B \cap C)}{\mathbb{P}(A^c\cap C)}=\mathbb{P}(B)\rightarrow\mathbb{P}(A^c\cap B \cap C)=\mathbb{P}(B)\mathbb{P}(A^c \cap C)=^{P1}\mathbb{P}(B)\mathbb{P}(A^c)\mathbb{P}(C)\ \checkmark$
$\frac{\mathbb{P}(A^c\cap B \cap C)}{\mathbb{P}(A^c\cap B)}=\mathbb{P}(C)\rightarrow\mathbb{P}(A^c\cap B \cap C)=\mathbb{P}(C)\mathbb{P}(A^c \cap B)=^{P1}\mathbb{P}(C)\mathbb{P}(B)\mathbb{P}(A^c)\ \checkmark$

- **Proposizione 4**: in generale se $A,B,C$ sono indipendenti, allora anche $A^c,B^c,C$ lo sono:
**Dim**: $A^c,B^c,C$ indipendenti $\iff \mathbb{P}(A^c|(B^c\cap C))=\mathbb{P}(A^c),\mathbb{P}(B^c|(A^c\cap C))=\mathbb{P}(B^c),\mathbb{P}(C|(B^c\cap A^c))=\mathbb{P}(C)$
$\frac{\mathbb{P}(A^c\cap B^c \cap C)}{\mathbb{P}(B^c\cap C)}=\mathbb{P}(A^c)\rightarrow\mathbb{P}(A^c\cap B^c \cap C)=\mathbb{P}(A^c)\mathbb{P}(B^c \cap C)=^{P1}\mathbb{P}(A^c)\mathbb{P}(B^c)\mathbb{P}(C)\ \checkmark$
$\frac{\mathbb{P}(B^c\cap A^c \cap C)}{\mathbb{P}(A^c\cap C)}=\mathbb{P}(B^c)\rightarrow\mathbb{P}(A^c\cap B^c \cap C)=\mathbb{P}(B^c)\mathbb{P}(A^c \cap C)=^{P1}\mathbb{P}(B^c)\mathbb{P}(A^c)\mathbb{P}(C)\ \checkmark$
$\frac{\mathbb{P}(B^c\cap A^c \cap C)}{\mathbb{P}(A^c\cap B^c)}=\mathbb{P}(C)\rightarrow\mathbb{P}(A^c\cap B^c \cap C)=\mathbb{P}(C)\mathbb{P}(A^c \cap B^c)=^{P2}\mathbb{P}(C)\mathbb{P}(A^c)\mathbb{P}(B^c)\ \checkmark$

- **Proposizione 5**: in generale se $A,B,C$ sono indipendenti, allora anche $A^c,B^c,C^c$ lo sono:
**Dim**: $A^c,B^c,C^c$ indipendenti $\iff \mathbb{P}(A^c|(B^c\cap C^c))=\mathbb{P}(A^c),\mathbb{P}(B^c|(A^c\cap C^c))=\mathbb{P}(B^c),\mathbb{P}(C^c|(B^c\cap A^c))=\mathbb{P}(C^c)$
$\frac{\mathbb{P}(A^c\cap B^c \cap C^c)}{\mathbb{P}(B^c\cap C^c)}=\mathbb{P}(A^c)\rightarrow\mathbb{P}(A^c\cap B^c \cap C^c)=\mathbb{P}(A^c)\mathbb{P}(B^c \cap C^c)=^{P2}\mathbb{P}(A^c)\mathbb{P}(B^c)\mathbb{P}(C^c)\ \checkmark$
$\frac{\mathbb{P}(A^c\cap B^c \cap C^c)}{\mathbb{P}(A^c\cap C^c)}=\mathbb{P}(B^c)\rightarrow\mathbb{P}(A^c\cap B^c \cap C^c)=\mathbb{P}(B^c)\mathbb{P}(A^c \cap C^c)=^{P2}\mathbb{P}(B^c)\mathbb{P}(A^c)\mathbb{P}(C^c)\ \checkmark$
$\frac{\mathbb{P}(A^c\cap B^c \cap C^c)}{\mathbb{P}(A^c\cap B^c)}=\mathbb{P}(C^c)\rightarrow\mathbb{P}(A^c\cap B^c \cap C^c)=\mathbb{P}(C^c)\mathbb{P}(A^c \cap B^c)=^{P2}\mathbb{P}(C^c)\mathbb{P}(B^c)\mathbb{P}(A^c)\ \checkmark$

La **(1),(2),(3)** sono verificate date queste **5** proposizioni.

**Comment by daromacarlo** [comment]
punto1) 
pubblicato perchè ho una domanda. Posso porre A (complementare) = C (unito) B?
Se si, è giusto il agionamento che ho fatto?
Grazie.

![Immagine 2023-10-26 182900](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/135441131/e7599376-c15b-4da6-849a-2b6a05285e1e)

**Comment by Elia-Belli** [comment]
@CarloDaRomadev due eventi indipendenti non sono necessariamente disgiunti, quindi non è detto che $A^c$ sia come l hai definito e che quello sia il tuo diagramma di Venn

**Comment by daromacarlo** [comment]
@Elia-Belli 👍

**Comment by daromacarlo** [comment]
@Elia-Belli quando hai tempo potresti spiegarmi i passaggi, non riesco a seguire il ragionamento, grazie.

**Comment by daromacarlo** [comment]
il tuo punto 1 l'ho capito così ma penso di sbagliare qualcosa.

![Immagine 2023-11-09 192032](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/135441131/5b225d64-87a6-4ed2-9974-a8250a2dbdaa)

**Comment by cornflexxx** [comment]
Per dimostrare tutte e tre le proposizioni è sufficiente dimostrare che sono indipendenti a coppie, in generale  possiamo dimostrare che se $A$ e $B$ sono indipendenti allora $A^c$ e $B$ sono indipendenti, infatti:

$P(B)=P(BA)\cdot P(BA^c)=P(B)P(A)\cdot P(BA^c)$
$P(B)-P(B)P(A)=P(BA^c)$
$P(B)-P(B)(1-P(A^c))=P(BA^c)$
$P(B)P(A^c)=P(BA^c)$
